tardis/montecarlo/packet_source.py

```diff
@@ -40,14 +40,14 @@ def create_packet_nus(self, T, no_of_packets):
 
         Parameters
         ----------
-        T : ~float
+        T : float
             temperature
-        no_of_packets: ~int
+        no_of_packets: int
 
         Returns
         -------
 
-            : ~numpy.ndarray
+            : numpy.ndarray
             array of frequencies
         """
```

Comment:
The indentation here is wrong, also no need for the leading `:` if there is no name for the return value but only a type. There are more similar instances.